fleet/lib/opensearch-logging/fleet.yaml

```diff
@@ -0,0 +1,45 @@
+---
+defaultNamespace: logging
+namespaceLabels:
+  lsst.io/discover: "true"
+  lsst.io/author: "fbegyn"
+labels:
+  bundle: &name logging
+helm:
+  releaseName: *name
+  takeOwnership: true
+  force: true
+  waitForJobs: true
+  atomic: false
+dependsOn:
+  - selector:
+      matchLabels:
+        bundle: opensearch-operator
```

Comment:
Please add a dep on `cert-manager-conf`.

Reply:
added the depency on `cert-manager-conf` in [42debf8](https://github.com/lsst-it/k8s-cookbook/pull/231/commits/42debf8ed65d6a40977706d293c34890c76df0b7)